Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.8.12
+# not sure what these two lines do but they were in
+# an example I found.
+ENV PYTHONDONTWRITEBYTECODE=1
```

Comment:
It seems setting this flag stops bytecode (pyc) files from getting created and ensures that we don't actually run stale python modules if the .py files are newer.

I've been bitten by this before so seems the only harm is reduced performance which doesn't seem like something we need to be concerned with right now.